src/components/dialog/examples/dialog-form.tsx

```diff
@@ -75,31 +75,27 @@ export class DialogFormExample {
                         <limel-slider unit="%" value={this.percentage} />
                     </p>
                 </form>
-                <limel-flex-container slot="button" reverse={true}>
+                <limel-flex-container slot="button">
+                    <limel-button label="Cancel" onClick={this.closeDialog} />
                     <limel-button
                         primary={true}
                         label="Save"
                         disabled={!this.nameValid() || !this.ageValid()}
                         onClick={this.submitForm}
                     />
-                    <limel-button label="Cancel" onClick={this.closeDialog} />
```

Comment:
> Having this was completely unnecessary from the beginning.
> Not sure why it was added in the example?
> Maybe to showcase what `flex-container` can do?!
> But consumers copy/paste this exact code
> and it potentially makes troubles for us.
> For example when trying to add fixes such as the previous commit.
> Which will cause the buttons to jump to the left side of the dialog,
> if consumer has copy/pasted `reverse={true}` as well.

The idea was that it would be easier for the users if a single tab-press brought them to the primary button, instead of having to tab past the cancel-button first.

It was probably misguided… 🤷‍♂️

Reply:
I see... I'm a bad example and I no clue about how people navigate with keyboard. But I think their expectation is to navigate from left to right. So I'd suggest we skip this, unless we have some usability/accessibility guidelines that says this is the expected behaviour.

Reply:
Yeah, I agree. I don't have any guidelines for it, just an idea I had before I knew better 😆

Kill it with fire! 🙂